dashboard/src/app/workspaces/workspace-details/environments/environments.directive.js

```diff
@@ -40,7 +40,7 @@ export class WorkspaceEnvironments {
 
     this.scope = {
       environmentName: '=',
-      environmentViewStatus: '=',
+      machinesViewStatus: '=',
```

Comment:
I don't think so. I think that prefixes are needed for our customized stuff to distinguish from standard/third-party ones.